src/docs/user/references/Notification-service.adoc

```diff
@@ -0,0 +1,237 @@
+=== Overview
+
+The <<_glossary_notification_portal,*Notification Portal*>> allows users to subscribe, be notified and react when different
+types of events occur during <<_glossary_job,*Job*>> execution, on the state of the <<_glossary_proactive_scheduler,*ProActive Scheduler*>> or when
+automating and scheduling jobs with the <<_glossary_job_planner,*Job Planner*>>.
+
+image::notifications-view.png[align="center"]
+
+=== Subscriptions
+
+==== Principle
+
```

Comment:
this section should be improved and seems a bit short to me.

You need to answer the question, what is a subscription and what does it consist of.

Example of what you can add:

A notification subscription consists in a set of events received by a user for a particular service or a particular type of workflows. Event reception is configurable and allows portal notifications, email sending, custom script execution, etc.

A user creating a subscription will need to:
 - select one of the subscription types
 - select one or more events that will be received.  
 - select one or more notification methods.